### PR TITLE
Add empty block test

### PR DESCRIPTION
**Pull request by kamilsa**
Signed-off-by: kamilsa <kamilsa16@gmail.com>

### Description of the Change

Add test on empty block
### Benefits

Empty block behaviour is tested

### Possible Drawbacks 

This test might be meaningless, because other tests can test the same, although they are not targeting this purpose

### Usage Examples or Tests 

empty_block_pipeline_test.cpp

### CODE REVIEW

File: test/integration/pipeline/empty_block_pipeline_test.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#include <gtest/gtest.h>
+
+#include "builders/protobuf/transaction.hpp"
+#include "framework/batch_helper.hpp"
+
+const shared_model::crypto::Keypair keypair =
+    shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair();
+
+/**
+ * prepares signed transaction with CreateDomain command
```
Comment (Akvinikym) [inline test/integration/pipeline/empty_block_pipeline_test.cpp]: I think, you should reflect in docs that it prepares **invalid** transaction

File: test/integration/pipeline/empty_block_pipeline_test.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      .createDomain(domain_name, "user")
+      .build()
+      .signAndAddSignature(keypair)
+      .finish();
+}
+
+/**
+ * prepares transaction sequence
+ * @param tx_size the size of transaction sequence
+ * @return  transaction sequence
```
Comment (Akvinikym) [inline test/integration/pipeline/empty_block_pipeline_test.cpp]: Double space between first two words

File: test/integration/pipeline/empty_block_pipeline_test.cpp [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+ */
+auto prepareTransactionSequence(size_t tx_size) {
```
Comment (Akvinikym) [inline test/integration/pipeline/empty_block_pipeline_test.cpp]: `prepareInvalidTransactionSequence`?

File: test/integration/pipeline/empty_block_pipeline_test.cpp [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+/**
+ * @given set of stateful invalid transactions
+ * @when all transactions are sent to iroha
+ * @then verified proposal is empty and no block is committed
+ */
+TEST(EmptyBlockPipelineIntegrationTest, SendInvalidSequence) {
+  size_t tx_size = 5;
+  const auto &tx_sequence = prepareTransactionSequence(tx_size);
+
+  auto check_block = [](auto &block) {
```
Comment (Akvinikym) [inline test/integration/pipeline/empty_block_pipeline_test.cpp]: `check_proposal`, actually

File: test/integration/pipeline/empty_block_pipeline_test.cpp [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    ASSERT_EQ(block->transactions().size(), 0);
+  };
+  integration_framework::IntegrationTestFramework(
+      tx_size)  // make all transactions to fit into a single proposal
+      .setInitialState(keypair)
+      .sendTxSequence(tx_sequence, [](const auto &) {})
+      .checkProposal([](const auto proposal) {})
```
Comment (Akvinikym) [inline test/integration/pipeline/empty_block_pipeline_test.cpp]: You can just `skipProposal()`

**Comment by kamilsa** [comment]
This PR is not needed anymore, since EmptyBlock entoty will be removed
